muzero.py

```diff
@@ -35,8 +35,9 @@ def __init__(self, game_name):
         # Load the game and the config from the module with the game name
         try:
             game_module = importlib.import_module("games." + self.game_name)
+            getGameClassName = getattr(game_module, 'getGameClassName')
+            self.Game = getattr(game_module, getGameClassName())
```

Comment:
I think we can avoid this by renaming the abstract class "AbstractClass". The game class name will still be Game and we will not need a getGameClassName parameter. I find it simpler.